Content.Server/Audio/Jukebox/JukeboxSystem.cs

```diff
@@ -153,4 +224,29 @@ private void TryUpdateVisualState(EntityUid uid, JukeboxComponent? jukeboxCompon
 
         _appearanceSystem.SetData(uid, JukeboxVisuals.VisualState, finalState);
     }
+
+    private JukeboxBoundUserInterfaceState GetUiState(Entity<JukeboxComponent> ent)
+    {
+        var isPlaying = false;
+        if (Exists(ent.Comp.AudioStream))
+        {
+            isPlaying = Audio.IsPlaying(ent.Comp.AudioStream);
+        }
```

Comment:
IsPlaying should already account for this and if it really doesn't then it should.

Reply:
Removed Exists check.